internal/servers/events_server.go

```
@@ -0,0 +1,317 @@
+		logger.DebugContext(ctx, "Canceled subcription")
+	}()
+
+	// Wait to receive events on the channel of the subscription and forward them to the client:
+	for {
+		select {
+		case event, ok := <-subInfo.eventsChan:
+			if !ok {
+				logger.DebugContext(ctx, "Subscription channel closed")
+				return nil
+			}
+			stream.Send(&eventsv1.EventsWatchResponse{
```

golangci-lint (1.64.8)

Error return value of `stream.Send` is not checked

(errcheck)